packages/babel-plugin-transform-wpcalypso-async/package.json

```diff
@@ -16,5 +16,8 @@
 	"devDependencies": {
 		"@automattic/calypso-typescript-config": "workspace:^",
 		"@babel/core": "^7.16.0"
+	},
+	"peerDependencies": {
+		"@babel/core": "^7.16.0"
```

Comment:
Why do we need both a peer and dev dependency for `@babel/core`?

Reply:
In general, `devDependencies` are the dependencies that you need when developing the package, but they won't be installed by consumers. `peerDepenencies` are requirements for consumers. So having the same dep in `dev` and `peer` is a way to satisfy your own `peerDependency` requirements. Does that make sense?

In this specific case, I don't remember why I added it. Probably was a transitive peerDependency from some dependency, but I'll check.

Reply:
Yep! makes sense 👍